spanner_orm/tests/update_test.py

```diff
@@ -100,6 +135,25 @@ def test_create_table_interleaved(self, get_model):
         )
         self.assertEqual(test_update.ddl(), test_model_ddl)
 
+    @mock.patch("spanner_orm.admin.metadata.SpannerMetadata.model")
+    def test_create_table_foreign_keys(self, get_model):
+        get_model.return_value = None
+        new_model = models.RelationshipTestModel
+        test_update = update.CreateTable(new_model)
+        test_update.validate()
+        print(test_update.ddl())
+
+        test_model_ddl = (
+            "CREATE TABLE RelationshipTestModel ("
+            "parent_key STRING(MAX) NOT NULL, "
+            "child_key STRING(MAX) NOT NULL, "
+            "CONSTRAINT parent FOREIGN KEY (parent_key) REFERENCES spanner_orm.tests.models.SmallTestModel (key), "
```

Comment:
maybe this is just bad test fixture, but should this be `SmallTestModel` instead of `spanner_orm.tests.models.SmallTestModel`

Reply:
Yeah maybe a bad test fixture ... I use the name specified as the `destination_handle` when instantiating the `Relationship` class and the `RelationshipTestModel` uses:
```python
parent = relationship.Relationship(
        "spanner_orm.tests.models.SmallTestModel", {"parent_key": "key"}, single=True
    )
```
I decided not to modify the `RelationshipTestModel` too much apart from adding the `fk_multicolumn` relationship.

In real practice, it definitely has to be the name of the table otherwise it will fail.